### DISCUSSION TITLE
[@types/three] Missing type definition for DebugEnvironment

### DISCUSSION THREAD

**Discussion by skyclouds2001** [DefinitelyTyped/DefinitelyTyped#66496] [Issues with a @types package] [answered]
DebugEnvironment has no type definition yet

[source folder](https://github.com/mrdoob/three.js/tree/master/examples/jsm/environments)

[target folder](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/three/examples/jsm/environments)

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
Hey @skyclouds2001!

Feel free to create a PR to fix this at https://github.com/three-types/three-ts-types. There are too many three.js examples for us to keep with them, so [we rely on the community to create PRs](https://github.com/three-types/three-ts-types#priorities-and-goals) to add type definitions for the examples that currently have no type definitions.

**Reply by skyclouds2001** [reply]
Got it!

**Comment by skyclouds2001** [comment, marked as answer]
See https://github.com/three-types/three-ts-types/pull/589
